cmd/snapshot-controller/main.go

```diff
@@ -228,10 +240,10 @@ func main() {
 		workqueue.NewItemExponentialFailureRateLimiter(*retryIntervalStart, *retryIntervalMax),
 		*enableDistributedSnapshotting,
 		*preventVolumeModeConversion,
-		*enableVolumeGroupSnapshots,
+		utilfeature.DefaultFeatureGate.Enabled(features.VolumeGroupSnapshot),
```

Comment:
VolumeGroupSnapshot needs to be disabled by default when moving to Beta initially.

Reply:
this just checks if it is enabled or not, but in features, default value is false only